jobs/buildhub/s3_inventory_to_kinto.py

```diff
@@ -64,6 +64,10 @@
     cast=Csv()
 ))
 
+LOG_LEVEL = config('LOG_LEVEL', default='INFO')
+
+STORE_DAILY_MANIFEST = config('STORE_DAILY_MANIFEST', default=False, cast=bool)
```

Comment:
I just needed this for local dev. It can be ignored.

Reply:
What I found was weird. 

I would run the `latest-inventory-to-kinto` once (takes 1-2 hours) and it downloads all the `md5hashmd5hashmd5hashmd5hashmd5hash.csv.gz` files to disk. But when I run it immediately after, the code would download the inventory manifest again and now it's finding it needs to download new .csv.gz files. All within a single day!